packages/contracts-bedrock/invariant-docs/ResourceMetering.md

```diff
@@ -1,42 +1,2 @@
 # `ResourceMetering` Invariants
 
-## The base fee should increase if the last block used more than the target amount of gas
-**Test:** [`ResourceMetering.t.sol#L180`](../contracts/test/invariants/ResourceMetering.t.sol#L180)
-
-If the last block used more than the target amount of gas (and there were no empty blocks in between), ensure this block's baseFee increased, but not by more than the max amount per block. 
-
-
```

Comment:
Where did these go?

Reply:
Oh good catch - they really went like this
![image](https://github.com/ethereum-optimism/optimism/assets/21288394/e40b9417-ff98-492a-bd93-6b1dc63bad8e)

Reply:
Found the issue - working on a fix.

Reply:
Fixed.